crates/stdsimd-test/src/lib.rs

```diff
@@ -16,18 +16,23 @@ extern crate cc;
 extern crate lazy_static;
 extern crate rustc_demangle;
 extern crate simd_test_macro;
-extern crate wasm_bindgen;
-
-use std::collections::HashMap;
-use std::env;
-use std::path::Path;
-use std::process::Command;
-use std::str;
-
-use wasm_bindgen::prelude::*;
+#[macro_use]
+extern crate cfg_if;
 
 pub use assert_instr_macro::*;
 pub use simd_test_macro::*;
+use std::{collections::HashMap, env, str};
+
+cfg_if! {
+    if #[cfg(target_arch = "wasm32")] {
+        extern crate wasm_bindgen;
+        mod wasm;
```

Comment:
Yeah unfortunately with wasm-bindgen today you'll have to use `pub mod wasm` here instead of just `mod wasm`